app/assets/javascripts/table-sort.es6

```diff
@@ -38,17 +40,22 @@ $(document).ready(function() {
             // parentNode is the element we want to move
             return this.parentNode;
           });
-          
+
           //this adds a message to the message div, stating what it is sorted by
           $("#screen-reader-messages").html("Sorted by " + th.find(".aria-sort-description").text() + " " + endText[sortDirection][th.attr("data-sort-type")]);
-      
+          $(".screen-reader-sort-indicator").html("");
+          th.find(".screen-reader-sort-indicator").html(" (currently sorted " + endText[sortDirection][th.attr("data-sort-type")] + ").");
+
           inverse = !inverse;
-          
+
           return false;
         });
-        th.keyup(function(ev) {
-          if (ev.which==13 || ev.which==32)  { //on space or return, the column is sorted
+        th.keypress(function(ev) {
+          const spaceBarCode = 32;
+          const returnKeyCode = 13;
+          if (ev.which==returnKeyCode || ev.which==spaceBarCode)  { //on space or return, the column is sorted
             $(this).click();
+            return false;
```

Comment:
This is the same code as is used in `app/assets/javascripts/table-select-all.es6` lines 30-35. Looks like a common function could be used.